oci/private/tarball.sh.tpl

```diff
@@ -48,9 +49,7 @@ if [[ "${FORMAT}" == "oci" ]]; then
   # Handle multi-architecture image indexes.
   # Ideally the toolchains we rely on would output these for us, but they don't seem to.
 
-  layout_file=$(mktemp)
-  echo -n '{"imageLayoutVersion": "1.0.0"}' > "$layout_file"
-  add_to_tar "$layout_file" oci-layout
+  add_to_tar "{{oci_layout}}" oci-layout
```

Comment:
the oci_image that's being added to the tarball already has a oci-layout file in, we should use that instead.

Reply:
Can you tell me what change to make then? Do we need to copy it from somewhere? just delete this `add_to_tar`?

Reply:
i think you just need to change "{{oci_layout}}" to "${IMAGE_DIR}/oci-layout"